src/Lykke.Service.HFT/Controllers/OrderBooksController.cs

```diff
@@ -58,7 +60,24 @@ public OrderBooksController(IOrderBooksService orderBooksService, IAssetServiceD
             }
 
             var orderBooks = await _orderBooksService.GetAsync(assetPairId);
-            return Ok(orderBooks);
+            return Ok(orderBooks.Select(ToModel));
+        }
+
+        private static OrderBookModel ToModel(OrderBook orderBook)
```

Comment:
This looks like wasting of CPU time. Can we return the same model here, without mapping?